### PR TITLE
Simplify web project templates #10347

### PR DESCRIPTION
**Pull request by isaacrlevin**
Summary of the changes

- Remove CDNs from all Web Projects
- Remove environment tags from templates

Addresses #10347

### CODE REVIEW

**Comment by isaac2004** [comment]
Let me know if you want me to remove CDN tests here

https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/test/CdnScriptTagTests.cs

File: src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Shared/_ValidationScriptsPartial.cshtml [hunk 1]
```diff
@@ -1,18 +1,5 @@
...
-            asp-fallback-test="window.jQuery && window.jQuery.validator && window.jQuery.validator.unobtrusive"
-            crossorigin="anonymous"
-            integrity="sha256-9GycpJnliUjJDVDqP0UEu/bsm9U+3dnQUH8+3W10vkY=">
-    </script>
-</environment>
+﻿
+<script src="~/lib/jquery-validation/dist/jquery.validate.js"></script>
+<script src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js"></script>
+
+
```
Comment (ryanbrandenburg) [inline src/ProjectTemplates/Web.ProjectTemplates/content/RazorPagesWeb-CSharp/Pages/Shared/_ValidationScriptsPartial.cshtml]: Nit-pick: Extra newline here.

**Review by ryanbrandenburg — APPROVED** [review]
I think you're good to go once you remove the CDN tests.

**Comment by ryanbrandenburg** [comment]
> Let me know if you want me to remove CDN tests here
> 
> https://github.com/aspnet/AspNetCore/blob/master/src/ProjectTemplates/test/CdnScriptTagTests.cs

Yeah, they no longer serve a purpose so go ahead and remove them.

**Comment by isaac2004** [comment]
@ryanbrandenburg do I need a merge from main here? Test failed in a weird spot

**Comment by isaac2004** [comment]
@ryanbrandenburg doing a merge does not resolve CI

**Comment by ryanbrandenburg** [comment]
@aspnet-hello triage https://dev.azure.com/dnceng/public/_build/results?buildId=211914

**Comment by aspnet-hello** [comment]
This comment was made automatically. If there is a problem contact aspnetcore-build@microsoft.com.

I've triaged the above build. I've created/commented on the following issue(s)
https://github.com/aspnet/AspNetCore-Internal/issues/2450
https://github.com/aspnet/AspNetCore-Internal/issues/2614
https://github.com/aspnet/AspNetCore-Internal/issues/2572
https://github.com/aspnet/AspNetCore-Internal/issues/2626

**Comment by ryanbrandenburg** [comment]
All the failures were flaky tests, I'm re-running it to see if that cleans things up.

**Pull request merged**